### PR TITLE
Add built-in function sqrt(#2097)

### PR DESCRIPTION
**Pull request by caaatch22**
**What type of PR is this?**

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [ ] Documentation
- [x] Feature
- [ ] Test and CI
- [ ] Code Refactoring

**Which issue(s) this PR fixes:**

issue #2097 

**What this PR does / why we need it:**

Add built-in function sqrt

**Special notes for your reviewer:**

Not Available

**Additional documentation (e.g. design docs, usage docs, etc.):**

Not Available

### CODE REVIEW

File: pkg/vectorize/sqrt/sqrt.go [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+func sqrtUint16(xs []uint16, rs []float64) SqrtResult {
+	result := SqrtResult{Result: rs, Nsp: new(nulls.Nulls)}
+	for i, n := range xs {
+		result.Result[i] = math.Sqrt(float64(n))
+	}
+	return result
+}
+
+func sqrtUint32(xs []uint32, rs []float64) SqrtResult {
```
Comment (broccoliSpicy) [inline pkg/vectorize/sqrt/sqrt.go:70]: we are switching to Golang generics now, wanna check it out [go generics](https://go.dev/doc/tutorial/generics)

sorry our tutorial didn't say about generics

**Review by broccoliSpicy — CHANGES_REQUESTED** [review]

File: pkg/vectorize/sqrt/sqrt.go (hunk 1 above)
Comment (caaatch22) [reply pkg/vectorize/sqrt/sqrt.go:70]: ok, I’ll modify it

**Review by broccoliSpicy — APPROVED** [review]
looks good to me

**Comment by fengttt** [comment]
Close to free resource on ci.

**Pull request closed without merging**